tests/lightning_qubit/test_state_vector_class.py

```diff
@@ -24,25 +24,56 @@
 from pennylane.tape import QuantumScript
 from pennylane.wires import Wires
 
-from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
+if device_name == "lightning.qubit":
+    from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
+
+if device_name == "lightning.kokkos":
+    from pennylane_lightning.lightning_kokkos._state_vector import (
+        LightningKokkosStateVector as LightningStateVector,
+    )
+
+    try:
+        from pennylane_lightning.lightning_kokkos_ops import InitializationSettings
+    except ImportError:
+        pass
 
-if device_name != "lightning.qubit":
-    pytest.skip("Exclusive tests for lightning.qubit. Skipping.", allow_module_level=True)
+
+if device_name not in ("lightning.qubit", "lightning.kokkos"):
+    pytest.skip(
+        "Exclusive tests for lightning.qubit or lightning.kokkos. Skipping.",
+        allow_module_level=True,
+    )
 
 if not LightningDevice._CPP_BINARY_AVAILABLE:
     pytest.skip("No binary module found. Skipping.", allow_module_level=True)
 
 
 @pytest.mark.parametrize("num_wires", range(4))
 @pytest.mark.parametrize("dtype", [np.complex64, np.complex128])
-@pytest.mark.parametrize("device_name", ["lightning.qubit"])
-def test_device_name_and_init(num_wires, dtype, device_name):
+def test_device_name_and_init(num_wires, dtype):
     """Test the class initialization and returned properties."""
     state_vector = LightningStateVector(num_wires, dtype=dtype, device_name=device_name)
     assert state_vector.dtype == dtype
     assert state_vector.device_name == device_name
     assert state_vector.wires == Wires(range(num_wires))
 
+    if device_name == "lightning.kokkos":
+        bad_kokkos_args = np.array([33])
+        with pytest.raises(
+            TypeError,
+            match=f"Argument kokkos_args must be of type {type(InitializationSettings())} but it is of {type(bad_kokkos_args)}.",
+        ):
+            assert LightningStateVector(
+                num_wires, dtype=dtype, device_name=device_name, kokkos_args=bad_kokkos_args
+            )
+
+        set_kokkos_args = InitializationSettings().set_num_threads(2)
+        state_vector_3 = LightningStateVector(
+            num_wires, dtype=dtype, device_name=device_name, kokkos_args=set_kokkos_args
+        )
+
+        assert type(state_vector) == type(state_vector_3)
+
```

Comment:
Cover different options of initialization for LKokkos

Reply:
Nice job!